server/constants/requestMethods.ts

```diff
@@ -1,17 +1,23 @@
 import { ApplicationMethodType } from '../data/orchestrationApiTypes'
 
-export const requestMethodOptions: Readonly<Partial<Record<ApplicationMethodType, string>>> = {
+export const requestMethodsCancellation: Readonly<Partial<Record<ApplicationMethodType, string>>> = {
```

Comment:
I've made two sets of request methods to support different lists for booking / cancellation